docs/articles/other/Enable-Location-Access-on-Web.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Enable Location Access on Web
+description: How to enable location access for Expensify websites on your browser
+---
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# About
+<!--
+This short blurb provides all the context the user might need to know what the feature is and why they'd use it. If there are particular best practices embedded in the design, we'd explain why they're best practices here.
+
+What advantages does this feature provide in general?
+What is unique about this feature in terms of our specific implementation of it in Expensify?
+-->
+If you'd like to use features that rely on your current location you will need to enable location permissions for Expensify. You can find instructions for how to enable location settings on the three most common web browsers below. If your browser is not in the list then please do a web search for your browser and "enable location settings".
+
+# How-to
+<!--
+This section covers the most essential information a user needs to operate a feature i.e. what to click on. We’ll go over any action the user might take when configuring or using the feature, starting from configuration and moving to usage.
+
+What options does a user have when configuring this feature?
+What options does a user have then interacting with this feature?
+What elements of this feature are pay-walled vs. free?
+-->
+
+### Chrome
+1. Open Chrome
+2. At the top right, click the three-dot Menu > Settings
+3. Click "Privacy and Security" and then "Site Settings"
+4. Click Location
+5. Check the "Not allowed to see your location" list to make sure expensify.com and new.expensify.com are not listed. If they are, click the delete icon next to them to allow location access
+
+[Chrome help page]([url](https://support.google.com/chrome/answer/142065))
```

Comment:
The markdown here isn't quite right for the link. The format should be like this.
```
[Chrome help page](https://support.google.com/chrome/answer/142065)
```
 You can test this by pasting it in a GH comment and looking at the Markdown preview.
 
 But actually, let's make the link open in a new tab by default. I think that's more handy for this kind of thing. So the format is like this. I found that format [here](https://talk.jekyllrb.com/t/how-can-i-ensure-my-links-open-in-a-new-tab/4318) from a google search.
 ```
 [Chrome help page](https://support.google.com/chrome/answer/142065){:target="_blank"}
```

Reply:
All links updated with the correct format